build-site.py

```diff
@@ -34,7 +45,7 @@
     # Prepare data and build main website.
     version = helper.thunderbird_desktop.latest_version('release')
     beta_version = helper.thunderbird_desktop.latest_version('beta')
-    caldata = helper.load_calendar_json('media/caldata/calendars.json')
+    caldata = helper.load_calendar_json('media/caldata/autogen/calendars.json')
```

Comment:
Check if this file exists, and if not load the old one, because otherwise this will break builds on check-in.